pages.zh_TW/common/aws.md

```diff
@@ -0,0 +1,37 @@
+# aws
+
+> Amazon Web Services 官方的命令列介面工具。
+> 子命令（例如`aws s3`）有自己的說明文件。
```

Comment:
Refer to [subcommand-mention](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#zh_tw).
```suggestion
> 此命令也有關於其子命令的文件，例如：`aws s3`.
```